README.md

```diff
@@ -1,26 +1,109 @@
-# README
+# Rails Budget
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Rails Budget project is about a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+<img width="944" alt="imagen" src="https://user-images.githubusercontent.com/39852288/142979078-56246d00-cae3-4640-96eb-31019e9f7140.png">
 
-* Ruby version
+## Live Demo
 
-* System dependencies
+[Heroku](https://rails-budget.herokuapp.com/)
 
-* Configuration
+## Built With
 
-* Database creation
+- ![Ruby](https://img.shields.io/badge/Ruby-20232A?style=for-the-badge&logo=ruby&logoColor=61DAFB)
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+### Installation
 
-* Services (job queues, cache servers, search engines, etc.)
+1. Clone the repo
 
-* Deployment instructions
+   ```sh
+   git@github.com:sevinchek/budget-app.git
+   ```
 
-* ...
-# budget-app
-# budget-app
+### Usage
+
+1. Install ruby gems dependencies
+
+   ```sh
+   bundle install
+   ```
+
+2. Install node dependencies
+
+   ```sh
+   npm install
+   ```
+
+3. Database creation
+
+   ```
+   rails db:setup
+   ```
+
+4. Run the web server
+
+   ```sh
+   rails server
+   ```
+
+5. Access to the website in a browser with the following link
+
+   ```sh
+   localhost:3000
+   ```
+
+### Test
+
+1. For testing run:
+
+   ```sh
+   rspec spec/* --format doc
+   ```
+
+## Acknowledgements
+
+- Use ruby gems as software packages system.
+- Install Ruby on Rails framework.
+- Understand Rails RESTful design and router.
+- Use controllers to handle requests and render empty views.
+- Use params from browser request in a safe way.
+- Use preprocessed html file with embedded Ruby code.
+- Use layouts and templates for shared content.
+- Use database migration files to maintain database schema.
+- Use validations for models.
+- Secure app from n+1 problems.
+- Understand what ORM is.
+- Write SQL queries with ActiveRecord.
+- Set up associations between models.
+- Build a webapp that requires the user to log in.
+- Use devise gem for authentication.
+- Limit access to webapp resources based on authorization rules.
+- Analyze in writing why you have made a coding choice using one structure over another.
+
+## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
This issue has been addressed well. Good job! :clap: